src/Asset/EntrypointLookupInterface.php

```diff
@@ -9,10 +9,11 @@
 
 namespace Symfony\WebpackEncoreBundle\Asset;
 
+use Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface;
 use Symfony\Contracts\Service\ResetInterface;
 use Symfony\WebpackEncoreBundle\Exception\EntrypointNotFoundException;
 
-interface EntrypointLookupInterface extends ResetInterface
+interface EntrypointLookupInterface extends ResetInterface, WarmableInterface
```

Comment:
What’s the purpose of the WarmableInterface?

Reply:
This is to make sure the warmup method is there. I can remove the interface and just make a warmup method in this interface. Another thing is I can remove the interface and not have the warm up method here and the cache warmer can just assume the method exists.

Reply:
I think we should remove it - the warmer knows exactly what instances it’s workijg with and therefore what methods exist. What’s the purpose of this interface normally?

Reply:
The warmer knows about the collection and this interface. It does not know about the actual instance. I can have it do an instance check and on what is returned from the collection.